tests/metrics/lib/common.bash

```diff
@@ -0,0 +1,353 @@
+#!/bin/bash
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+THIS_FILE=$(readlink -f ${BASH_SOURCE[0]})
+LIB_DIR=${THIS_FILE%/*}
+RESULT_DIR="${LIB_DIR}/../results"
+
+source ${LIB_DIR}/../../lib/common.bash
+source ${LIB_DIR}/json.bash
+source /etc/os-release || source /usr/lib/os-release
+
+# Set variables to reasonable defaults if unset or empty
+CTR_EXE="${CTR_EXE:-ctr}"
+DOCKER_EXE="${DOCKER_EXE:-docker}"
+CTR_RUNTIME="${CTR_RUNTIME:-io.containerd.kata.v2}"
+RUNTIME="${RUNTIME:-containerd-shim-kata-v2}"
+KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
+TEST_REPO="${TEST_REPO:-github.com/kata-containers/tests}"
+JSON_HOST="${JSON_HOST:-}"
+
+KSM_BASE="/sys/kernel/mm/ksm"
```

Comment:
I remember the KSM values here and below were used by the ksm-throttler, which I believe is no longer supported, so it should be fine to delete them if we don't plan to test KSM, but would like to see if someone else can confirm.
For reference: https://github.com/kata-containers/kata-containers/issues/1073

Reply:
@chavafg we are not using ksm-throttler for Kata but we are using some memory tests that are only using ksm

Reply:
ok